### PR TITLE
Added image search

### PR DESCRIPTION
**Pull request by davidwavid**

### CODE REVIEW

File: DiscordBot/Managers/CommandManager_Methods.cs [hunk 1]
```diff
@@ -311,5 +311,16 @@ private async Task Time(DiscordSocketClient client, SocketMessage message, List<
             else
                 await message.Channel.SendMessageAsync($"UTC: {DateTime.UtcNow.ToString("yyyy-MM-dd HH:mm:ss")}");
         }
+
+        public async Task ImageSearch(DiscordSocketClient client, SocketMessage message, List<string> args)
+        {
+            var searchQuery = string.Join(' ', args);
+            if (string.IsNullOrWhiteSpace(searchQuery)) { 
+                await message.Channel.SendMessageAsync("You didn't provide an argument");
```
Comment (ijacab) [inline DiscordBot/Managers/CommandManager_Methods.cs:319]: You can throw badinputexception instead and it will be handled better

Comment (ijacab) [reply DiscordBot/Managers/CommandManager_Methods.cs:319]: Or badsyntaxexception is probably more fitting

**Review by scorttt — APPROVED** [review]
b

File: DiscordBot/Services/DuckDuckGoService.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using DiscordBot.Models.DuckDuckGoService;
+
+namespace DiscordBot.Services
+{
+    public class DuckDuckGoService
+    {
+        private static HttpClient _httpClient = new HttpClient();
```
Comment (ijacab) [inline DiscordBot/Services/DuckDuckGoService.cs:16]: U van add in the class constructor so DI handles it, see startup i did it for webalerter i tjink

**Review by ijacab — APPROVED** [review]

**Pull request merged**